packages/api/utils/ipLogging.test.ts

```diff
@@ -0,0 +1,9 @@
+import { IpCheckerMiddleware } from "./ipLogging";
+
+describe("Test console logging", function () {
+  it("should console log the event", async () => {
+    const consoleLogSpy = jest.spyOn(console, "log");
+    IpCheckerMiddleware();
+    expect(consoleLogSpy).toHaveBeenCalled();
```

Comment:
I did come across this test failing as well with `consoleLogSpy` not being called. I think this is because of how the middleware is being used with middy. A quick sample is below. You can also look at `xssSanitizer.test.ts` as an alternative to testing the middleware.

```ts
    const request: any = {
      ...validRequest,
      body: mockPortfolioStep,
      requestContext: { identity: { sourceIp: "10.2.2.2" } },
    };
    const baseHandler = (event: any) => {
      return event;
    };
    const handler = middy(baseHandler).use(IpCheckerMiddleware());
    await handler(request, {} as Context, () => null);
```